src/arc.ts

```diff
@@ -169,6 +257,11 @@ export async function initializeArc(provider?: any): Promise<boolean> {
     if (success) {
       initializedAccount = await _getCurrentAccountFromProvider(arc.web3);
 
+      const userContract = await biconomyLogin(initializedAccount, options);
+      if (userContract) {
+        alert(`User contract wallet address is ${userContract} Make sure to have reputation in this address to vote`);
+        console.log(`user contract wallet ${userContract}`);
```

Comment:
use of console statements should be minimized, and whenever used they must be marked by:

`// eslint-disable-next-line no-console`

Also, please use `console.error` and `console.warn` where appropriate, instead of `console.log`.

Reply:
Done